src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Notes;
+import seedu.address.model.person.Person;
+
+/**
+ * Deletes Notes from the person identified using it's displayed index from the address book.
+ */
+public class DeleteNotesCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete_notes";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the patient notes from the person identified by the index number used in the "
+            + "displayed person list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
```

Comment:
might want to specify that you should only delete notes from someone who has existing notes

Reply:
We will need to make our User Guide elaborate on these nitty gritty stuff too in the future